clients/admin-ui/src/pages/datastore-connection/[id].tsx

```diff
@@ -0,0 +1,67 @@
+import { Center, Spinner } from "@fidesui/react";
+import ConnectionsLayout from "datastore-connections/ConnectionsLayout";
+import type { NextPage } from "next";
+import { useRouter } from "next/router";
+import { useEffect } from "react";
+
+import { useAppDispatch, useAppSelector } from "~/app/hooks";
+import {
+  reset,
+  selectConnectionTypeState,
+  setConnection,
+  setConnectionOption,
+} from "~/features/connection-type";
+import { useGetDatastoreConnectionByKeyQuery } from "~/features/datastore-connections";
+import { ConnectionType } from "~/features/datastore-connections/constants";
+import EditConnection from "~/features/datastore-connections/edit-connection/EditConnection";
+
+import Custom404 from "../404";
+
+const EditDatastoreConnection: NextPage = () => {
+  const dispatch = useAppDispatch();
+  const router = useRouter();
+  const { connection, connectionOption, connectionOptions } = useAppSelector(
+    selectConnectionTypeState
+  );
+
+  const { data, isFetching, isLoading, isSuccess } =
+    useGetDatastoreConnectionByKeyQuery(router.query.id as string);
```

Comment:
Noticed that if I navigate directly to a connection's url the contents are blank:
![Screen Shot 2022-10-12 at 10 57 08](https://user-images.githubusercontent.com/2236777/195414636-c197e513-5890-44ac-87c3-743a4db05a98.png)

At first I assumed it was this query's fault, but there's this odd `/connection/undefined` query in the network tools:
![Screen Shot 2022-10-12 at 11 00 08](https://user-images.githubusercontent.com/2236777/195415161-1d402976-ad76-4bf9-bba6-5027a9c0f727.png)

Reply:
Resolving this issue now. Great catch!

Reply:
Issue has been resolved now.